install-on-cluster build typo?

Environment:
- Spark 3.2.0
- Scala 2.12.15

Running: 
```
make install-on-cluster HAIL_COMPILE_NATIVES=1 SCALA_VERSION=2.12.15 SPARK_VERSION=3.2.0
```
I get the error:
```BUILD SUCCESSFUL in 2m 5s
3 actionable tasks: 3 executed
cp -f build/libs/hail-all-spark.jar python/hail/backend/hail-all-spark.jar
rm -rf build/deploy
mkdir -p build/deploy
mkdir -p build/deploy/src
cp ../README.md build/deploy/
rsync -r \
    --exclude '.eggs/' \
    --exclude '.pytest_cache/' \
    --exclude '__pycache__/' \
    --exclude 'benchmark_hail/' \
    --exclude '.mypy_cache/' \
    --exclude 'docs/' \
    --exclude 'dist/' \
    --exclude 'test/' \
    --exclude '*.log' \
    python/ build/deploy/
# Clear the bdist build cache before building the wheel
cd build/deploy; rm -rf build; python3 setup.py -q sdist bdist_wheel
make: *** No rule to make target 'check-pip-lockfiles', needed by 'install-on-cluster'.  Stop.
```


Issue is fixed for me by renaming `install-on-cluster: $(WHEEL) check-pip-lockfiles` -> `install-on-cluster: $(WHEEL) check-pip-lockfile` on line 344 of hail/Makefile.

Many thanks,
Barney

--- a/hail/Makefile
+++ b/hail/Makefile
@@ -341,7 +341,7 @@ install: $(WHEEL)
 	hailctl config set query/backend spark
 
 .PHONY: install-on-cluster
-install-on-cluster: $(WHEEL) check-pip-lockfiles
+install-on-cluster: $(WHEEL) check-pip-lockfile
 	sed '/^pyspark/d' python/pinned-requirements.txt | grep -v -e '^[[:space:]]*#' -e '^$$' | tr '\n' '\0' | xargs -0 $(PIP) install -U
 	-$(PIP) uninstall -y hail
 	$(PIP) install $(WHEEL) --no-deps

